version/10_12_0.sh

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash -e
 
-NODE_VERSION=10.12.0
+NODE_VERSION=10.13.0
```

Comment:
The filename doesn't match the version anymore. Maybe we should keep the filenames generic (`10_x.sh` or something like that?)

Reply:
Sorry, i will update